docs/CONTRIBUTING.md

```diff
@@ -13,62 +13,68 @@ Here are some tips to make sure your Pull Request (PR) can be merged smoothly:
 
 1. [Install Pre-requisites](#pre-requisites)
 1. [Clone](#clone-the-repo)
-1. [Get your own API key](#get-your-own-api-key)
 1. [Start Dev Server](#start-dev-server)
+1. [Get your own API key](#get-your-own-api-key)
 1. [Enter API credentials](#enter-api-credentials)
 
 ### Pre-requisites
 
 * Install [Git](https://git-scm.com/downloads)
 * Install [NodeJS](https://nodejs.org/).
-* Install [Yarn](https://yarnpkg.com/en/docs/install). If you're used to NPM, see "[Migrating from NPM](https://yarnpkg.com/lang/en/docs/migrating-from-npm/)".
-* Windows-based developers will need to install `windows-build-tools` (`yarn global add windows-build-tools`) globally prior to running `yarn install`. Refer to issue #1439 for [details](https://github.com/DestinyItemManager/DIM/issues/1439).
-* It is highly recommended to use [VSCode](https://code.visualstudio.com/) to work on DIM. When you open DIM in VSCode, accept the recommended plugins it suggests.
+* Install [Yarn](https://yarnpkg.com/en/docs/install). Use Yarn 1.x as DIM is not guaranteed to be compatible with later versions of Yarn. If you're used to NPM, see "[Migrating from NPM](https://yarnpkg.com/lang/en/docs/migrating-from-npm/)".
+* Windows-based developers will need to install `windows-build-tools` (run `yarn global add windows-build-tools` in your terminal) globally prior to running `yarn install`. Refer to issue #1439 for [details](https://github.com/DestinyItemManager/DIM/issues/1439).
+* It is highly recommended to use [VSCode](https://code.visualstudio.com/) to work on DIM. When you open DIM in VSCode, accept the recommended plugins it suggests (find them manually by searching "@recommended" in the Extensions window).
 * Linux-based developers will need to install `build-essential` (`sudo apt-get install -y build-essential`) prior to running `yarn install`.
 
 **Docker Development**
 * As an alternative to installing the above, you can try a docker-based development environment, but be aware that none of the core developers use it so it may be broken. See: [Docker Development Guide](Docker.md)
 
-
 ### Clone the repo
 
-To locally **run a copy** of DIM, you can simply clone the code repository:
+If you only want to **run a copy** of DIM locally and are not interested in contributing changes to the project, you can directly clone the original repository to your local machine using
```

Comment:
Not sure I understand the intent behind this change. You need to check out and run the app locally whether you want to contribute or not. You can choose to fork later if you want, and can switch the remote to point to your fork or add another remote.

Reply:
I've changed this line back to the original wording. My understanding of git workflows is still not perfect, thanks for explaining!